### PR TITLE
fix(plugin-container-syntax): typical markdown blockquote syntax throwing error

### LINKED ISSUE #1489: [Bug]: Cannot read properties of undefined (reading 'toString') when `markdown.mdxRs: false`
### Details

related PR: #1469 

If we use Blockquotes and Links together and set `markdown.mdxRs: false`

```mdx
> [foo](bar)
```

error will occured

Image

### Reproduce Steps

1. open arbitrary rspress projects
2. write in mdx file
```mdx
> [foo](bar)
```
3. `pnpm run dev`

### PR DESCRIPTION
**Pull request by ArnavK-09**
## Summary
resolves #1489 

image

## Related Issue
#1469 

## Checklist

- [x] Tests updated (or not required).
- [ ] Documentation updated (or not required).

### CODE REVIEW

**Comment by ArnavK-09** [comment]
@Timeless0911 hey can you help me pls ;-;
something wrong with nx or pnpm

**Comment by Timeless0911** [comment]
You should run `corepack enable` in your Rspress project, and automatically use correct version of pnpm

image

Current lockfile should be reset first.

File: e2e/fixtures/github-alert/package.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "name": "@rspress-fixture/rspress-doc-template",
```
Comment (Timeless0911) [inline e2e/fixtures/github-alert/package.json:2]:
```suggestion
  "name": "@rspress-fixture/github-alert",
```

File: e2e/fixtures/github-alert/rspress.config.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import * as path from 'node:path';
+import { defineConfig } from 'rspress/config';
+
+export default defineConfig({
+  root: path.join(__dirname, 'doc'),
+  markdown: {
+    mdxRs: false,
+  },
+  themeConfig: {
```
Comment (Timeless0911) [inline e2e/fixtures/github-alert/rspress.config.ts:9]: can delete this field

Comment (ArnavK-09) [reply e2e/fixtures/github-alert/rspress.config.ts]: i hope you mentioned themeConfig, i deleted that one

**Comment by Timeless0911** [comment]
I have reset the lockfile, and you have forgot to write tests in https://github.com/ArnavK-09/rspress/tree/main/e2e/tests. 

This PR is able to merge, tests can be added later.

**Review by Timeless0911 — APPROVED** [review]

**Pull request merged**